setup.py

```diff
@@ -25,6 +25,14 @@
     ]
 }
 
+sources = {
+    "esm": "esm",
+    "esm.model": "esm/model",
+    "esm.inverse_folding": "esm/inverse_folding",
+    "esm.esmfold.v1": "esm/esmfold/v1",
+    "esm.scripts": "scripts"
```

Comment:
@nikitos9000 shall we move `scripts` under `esm` to make things more consistent? Only we don't want to move `scripts/atlas` directory; we could move `atlas` to be at root level..
Or can we just not have `esm.scripts` in the `sources`, it seems unnecessary for the fairseq cli https://github.com/facebookresearch/fairseq/blob/main/setup.py

Reply:
I don't have any opinion... As I  have free time now, just let me know your preference...

Reply:
PS: without `esm.scripts` in `sources`, pip/python will be unable to find the cli scripts